src/wasm-lib/kcl/src/executor.rs

```diff
@@ -1844,20 +1880,22 @@ impl ExecutorContext {
         program: &crate::ast::types::Program,
         exec_state: &mut ExecState,
         body_type: BodyType,
-    ) -> Result<(), KclError> {
+    ) -> Result<Option<KclValue>, KclError> {
+        let mut last_expr = None;
         // Iterate over the body of the program.
         for statement in &program.body {
             match statement {
                 BodyItem::ExpressionStatement(expression_statement) => {
                     let metadata = Metadata::from(expression_statement);
-                    // Discard return value.
-                    self.execute_expr(
-                        &expression_statement.expression,
-                        exec_state,
-                        &metadata,
-                        StatementKind::Expression,
-                    )
-                    .await?;
+                    last_expr = Some(
+                        self.execute_expr(
+                            &expression_statement.expression,
+                            exec_state,
+                            &metadata,
+                            StatementKind::Expression,
+                        )
+                        .await?,
+                    );
                 }
                 BodyItem::VariableDeclaration(variable_declaration) => {
                     for declaration in &variable_declaration.declarations {
```

Comment:
Can we set `last_expr = None` in this case? I see that the parser enforces `ExpressionStatement` at the end, but TypeScript AST transformations won't, and I'd rather be safe.